### PR TITLE
Plasmapy

### PR DESCRIPTION
**Pull request by SolarDrew**
Recipe for [plasmapy](https://github.com/plasmapy/plasmapy).

### CODE REVIEW

File: recipes/plasmapy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record=record.txt
```
Comment (CJ-Wright) [inline recipes/plasmapy/meta.yaml]: Do you need the `--offline` and `--no-git`?

Comment (SolarDrew) [reply recipes/plasmapy/meta.yaml]: I thought I did, but I'll try it without again and see if it builds.

Comment (SolarDrew) [reply recipes/plasmapy/meta.yaml]: Turns out I do need the `--offline` but not the `--no-git`, so I'll remove that.

File: recipes/plasmapy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  skip: True  # [py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - astropy
+    - coveralls
```
Comment (bgruening) [inline recipes/plasmapy/meta.yaml]: this is usually not a requirement of a package, isn't it?

Comment (SolarDrew) [reply recipes/plasmapy/meta.yaml]: No, you're right, I think I put that in to try and solve some problem and then forgot to take it out when it didn't work.

**Comment by SolarDrew** [comment]
Hi @bgruening , @CJ-Wright , any further opinions on this?

File: recipes/plasmapy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
```
Comment (CJ-Wright) [inline recipes/plasmapy/meta.yaml]: Could you put the python version requirement into the deps and make this a noarch package? I'm assuming that this doesn't compile anything.

File: recipes/plasmapy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
```
Comment (CJ-Wright) [inline recipes/plasmapy/meta.yaml]: Could you use pip for this? (See the cf docs)

File: recipes/plasmapy/meta.yaml (hunk 3 above)
Comment (SolarDrew) [reply recipes/plasmapy/meta.yaml]: Not currently, no, so I've changed that. Thanks.

File: recipes/plasmapy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  noarch: python
```
Comment (msarahan) [inline recipes/plasmapy/meta.yaml]: Has compiled extensions.  Can't be noarch.  I'll submit a commit with it changed.

**Comment by wtbarnes** [comment]
I've pinged @SolarDrew about this. I believe `roman` was recently added as a dependency so this should be an easy fix.

**Comment by StanczakDominik** [comment]
This can be closed now! @SolarDrew ping :)

**Comment by StanczakDominik** [comment]
Re-pinging this PR as obsoleted by #6694 - let's get the open PR count down by 1, shall we?

**Pull request closed without merging**